test/fixtures/structured-data/suggestions.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "opportunityId": "opportunity-id",
+    "type": "CODE_CHANGE",
+    "rank": 2,
+    "data": {
+      "type": "url",
+      "url": "https://example.com/product/1",
+      "errors": [
+        "Missing field 'image'",
```

Comment:
Can we specify some `id` to the error? Asking because we want to show it in the dropdown and also to show the suggestions which are being specified.

Proposed structure :

```
{
  "type": "group",
  "name": "Group 1",
  "path": "https://site.url/*",
  "errors": [
    {
      "id": "error-1",
      "errorTitle": "Error 1",
      "fix": "<markdown-string>"
    },
    {
      "id": "error-2",
      "errorTitle": "Error 2",
      "fix": "<markdown-string>"
    }
  ]
}
```

Reply:
should this id be unique across the current opportunity?

Reply:
Yes it should be unique

Reply:
Can we auto-generate a random uuid and assign it here? I think that would work as well. I don't know the recommended way though 🤔

Reply:
yes I can generate a uuid and assign it for each error for each oppty